flight/mcu/lib/teensyShared/ActiveControl.h

```diff
@@ -1,41 +1,34 @@
 #include <PWMServo.h>
-#include <math.h>
 #include <rk4.h>
 
-#include <array>
-
 #include "ServoControl.h"
-#include "dataLog.h"
-
-using std::array;
 
 class Controller {
    public:
-    void ctrlTickFunction(pointers* pointer_struct);
+    void ctrlTickFunction();
     bool ActiveControl_ON();
-    Controller(struct pointers* pointer_struct, PWMServo* controller_servo);
+    Controller();
+    void init();
+
+    void startup();
 
     void setLaunchPadElevation();
 
-    PWMServo* controller_servo_;
-    mutex_t* dataMutex_state_;
-    stateData* stateData_;
+    PWMServo controller_servo_;
     rk4 rk4_;
     float kp = 0.00008;
 
-    float min_extension = -2.0 / 1000.0;
+    float min_extension = 0.0;
     float max_extension = 17.88 / 1000.0;
     float dt = .006;
     float prev_u = 0;
     float du_max = 0.01;
 
-    float launch_pad_alt;
-    float apogee_des_msl;
+    float launch_pad_alt{};
```

Comment:
Out of curiosity why are we doing the brackets around this. Is it so the default constructor gets called?

Reply:
Basically. They were there as an initial fix to ensure we explicitly initialize everything, though I'm going to go through and convert some of them to be explicitly initialized to 0.0